Makefile

```diff
@@ -0,0 +1,27 @@
+export PREFIX := /usr/
+
+SUBDIRS := utils/ data/default/
+
+.PHONY: all $(SUBDIRS)
```

Comment:
I find it stylistically nicer to have the `.PHONY` directly over the rule it qualifies, without a new line between them. But feel free to ignore me on that, it doesn't matter as long as they are related and close together..